lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status: s, response }, header } = this;
 
     // GET or HEAD for weak freshness validation only
-    if ('GET' != method && 'HEAD' != method) return false;
+    if (!['GET', 'HEAD'].includes(method)) return false;
```

Comment:
Knowing the array methods is not the problem. The name `includes` speaks for itself. However, at first glance, the first way conveys its meaning much better: "method is not GET and method is not HEAD", whereas the second one "GET and HEAD does not include method" takes a bit of consideration.

This is important when skimming through code while refactoring or adding a feature.